src/modules/roc_packet/target_stdio/roc_packet/address_to_str.cpp

```diff
@@ -18,28 +19,12 @@ address_to_str::address_to_str(const Address& addr) {
     buffer_[0] = '\0';
 
     switch (addr.version()) {
-    case 4: {
-        if (!addr.get_ip(buffer_, sizeof(buffer_))) {
-            roc_log(LogError, "address to str: can't format ip");
-        }
-
-        const size_t blen = strlen(buffer_);
-        if (snprintf(buffer_ + blen, sizeof(buffer_) - blen, ":%d", addr.port()) < 0) {
-            roc_log(LogError, "address to str: can't format port");
-        }
-
-        break;
-    }
+    case 4:
     case 6: {
```

Comment:
We can replace this switch with a single check `if (addr.valid())` to avoid duplicating this code.

Reply:
fixed